vite-project-kabinet/src/Content/PaginatedList.jsx

```diff
@@ -0,0 +1,53 @@
+import "./paginated.style.css"
+import { Card } from "./Card.jsx"
+import { useState } from "react";
+import arrowRight from "./assets/sipka_vpravo.svg"
+import arrowLeft from "./assets/sipka_vlevo.svg"
+
+export const PaginatedList = ({items, itemsPerPage}) => {
+    const [currentPage, setCurrentPage] = useState(1);
+
+    const totalPages = Math.ceil(items.length / itemsPerPage);
+
+    const handlePageChange = (newPage) => {
+        if (newPage >= 1 && newPage <= totalPages) {
+            setCurrentPage(newPage);
+        }
+    };
+
+    return (
+        <>
+            <section className="card__container">
+                {/* Zobrazení seznamu položek - tady namapuju komponentu <Card/> */}
+                {items
+                    .slice((currentPage - 1) * itemsPerPage, currentPage * itemsPerPage)
+                    .map((item) => (
+                    <Card key={item.id} genre={item.genre} title={item.title} author={item.author} content={item.content} image={item.image} />
+                ))}
+            </section>
```

Comment:
Tohle je správně a funguje to. Nicméně je dobrá praxe nemít moc dlouhé JS výrazy uvnitř JSX, protože se to trochu hůře čte. 
Zjednodušit se to dá takto:

```suggestion
const itemsPerPage = items.slice((currentPage - 1) * itemsPerPage, currentPage * itemsPerPage);
    return (
        <>
            <section className="card__container">
                {/* Zobrazení seznamu položek - tady namapuju komponentu <Card/> */}
                {itemsPerPage
                    .map((item) => (
                    <Card key={item.id} genre={item.genre} title={item.title} author={item.author} content={item.content} image={item.image} />
                ))}
            </section>
```